# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/moz.build
# Commit: 9d8f6e354d8c
# Full Hash: 9d8f6e354d8c332bab7e433252077ec0e1fa8f04
# Author: John Lin <jolin@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - remove MediaCodecVP8Codec. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115531
# ==============================================================================

diff -r 7675ead797c8 -r 9d8f6e354d8c dom/media/webrtc/libwebrtcglue/moz.build
--- a/dom/media/webrtc/libwebrtcglue/moz.build	Fri Oct 08 15:57:53 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/moz.build	Thu Oct 07 12:11:40 2021 -0700
@@ -31,10 +31,4 @@
     "WebrtcVideoCodecFactory.cpp",
 ]
 
-if CONFIG["OS_TARGET"] == "Android":
-    UNIFIED_SOURCES += [
-        "MediaCodecVideoCodec.cpp",
-        "WebrtcMediaCodecVP8VideoCodec.cpp",
-    ]
-
 FINAL_LIBRARY = "xul"